File: tdd-katas/video-store-kata/video-store-python/rental.py
class Rental:
	def __init__(self, movie, daysRented):
		self._movie 		= movie
		self._daysRented = daysRented
	
	def getDaysRented (self):
		return self._daysRented
	
	def getMovie (self):
		return self._movie

File: kata-solutions/video-store-kata/video-store-java/src/main/java/Rental.java
public class Rental {
	private final Movie movie;
	private final int daysRented;

  public Rental (final Movie movie, final int daysRented) {
		this.movie 		= movie;
		this.daysRented = daysRented;
	}
	
	public int getDaysRented () {
		return daysRented;
	}
	
	public Movie getMovie () {
		return movie;
	}

  public int determineFrequentRenterPoints() {
    return movie.determineFrequentRenterPoints(daysRented);
  }

  public double determineAmount() {
    return movie.determineAmount(daysRented);
  }
}